views/js/pciCreator/ims/mathEntryInteraction/helper/mathInPrompt.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+define(['core/moduleLoader'], function (moduleLoader) {
+    'use strict';
+
+    let MathJax;
+    let loaded = false;
+
+     /***
+     * lazy-load MathJax dependency
+     * @returns {Promise}
+     */
+    function load() {
+        if (loaded) {
+            return Promise.resolve();
+        }
+        return moduleLoader([], () => true)
+            .add( { module: 'mathJax', category: 'mathEntryInteraction'} )
+            .load()
+            .then(loadedModule => {
+                loaded = true;
+                MathJax = loadedModule && loadedModule.length ? loadedModule[0] : void 0;
+
+                // Do not wait between rendering each individual math element
+                // http://docs.mathjax.org/en/latest/api/hub.html
+                // https://github.com/oat-sa/tao-item-runner-qti-fe/blob/master/src/qtiCommonRenderer/renderers/Math.js
+                if (typeof MathJax !== 'undefined' && MathJax) {
+                    MathJax.Hub.processSectionDelay = 0;
+                }
+            });
+    };
+
+    const mathInPrompt = {
+        /**
+         * On dom element containing `<math>...</math>` MathML markup,
+         * run MathJax renderer ("typeset" it)
+         * @see https://github.com/oat-sa/tao-item-runner-qti-fe/blob/master/src/qtiCommonRenderer/renderers/Math.js
+         * @param {Object} $element
+         * @returns {Promise}
+         */
+        postRender: function postRender($element) {
+            if ($element.find('math').length === 0) {
+                return Promise.resolve();
+            }
+            return load().then(() => {
+                if (typeof MathJax !== 'undefined' && MathJax) {
```

Comment:
Why do you need the double check?
```suggestion
                if (MathJax) {
```

Reply:
You are right, changed. I copy-pasted it blindly from the common math renderer